### ISSUE TITLE
01/10/2017

### ISSUE THREAD

**Issue by azhe825** [ai-se/ZheYu#17]
## DOING:
1. Transfer learning in SLR

## DONE:
1. [tune lda to get a stable similarity score](https://github.com/ai-se/ML-assisted-SLR/issues/43)
2. [find a way to combine UPDATE and REUSE (therefore no need to decide which method to use before review)](https://github.com/ai-se/ML-assisted-SLR/issues/44)

## NOT DONE:
1. Drafting the paper. It is really really hard to do this in China, given the difficulty to look up literatures when google scholar is not available. Therefore I prioritized the experiments over writing. Will get back to writing jobs as soon as I am back to US.

## TODO:
1. Will decide if further experiments are necessary when full result of [UPDATE_REUSE](https://github.com/ai-se/ML-assisted-SLR/issues/43) comes out.
2. Some experiments on time decaying model.

## ADMIN
1. Any feedback on [UPDATE_REUSE](https://github.com/ai-se/ML-assisted-SLR/issues/43)?

**Comment by timm** [comment]
> China, given the difficulty to look up literatures when google scholar is not available

did not know that. amazing. nothing equivalent in china?

**Comment by azhe825** [comment]
not really. only thing we can do is to use really slow vpn to get to google scholar.

Anyway, I will make up the time lost on paper writing as soon as I get back to US
